scripts/backup_extractor.py

```diff
@@ -0,0 +1,23 @@
+import tarfile
+import os
+from pathlib import Path
+
+DOWNLOAD_DIR = Path(__file__).parent.parent / 'backups'
+TARGET_DIR = Path(__file__).parent.parent / 'db_backups'
+
+def extract_backups(path):
+     os.makedirs(TARGET_DIR, exist_ok=True)
```

Comment:
Lze pomocí https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir

Reply:
Tzn. pak by to vypadalo `TARGET_DIR.mkdir(exist_ok=True)` (možná i `parents=True`?)